soc/microchip/pic32cxsg/common/component/ac_component_fixup_pic32cxsg.h

```diff
@@ -0,0 +1,279 @@
+/*
+ * Copyright (c) 2024 Microchip
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _MICROCHIP_PIC32CXSG_AC_COMPONENT_FIXUP_H_
```

Comment:
Shouldn't this be at hal?

Reply:
I am just following the way that sam0/common was used.

Reply:
This is something that we should avoid always. We need to create specific commits on hal to make the correct adjustments. The commits should be component by component.